extra/sponge.importorder

```diff
@@ -0,0 +1,7 @@
+#Organize Import Order
```

Comment:
I don't know whether you just haven't uploaded it yet, but there are some classes/files with empty lines in the imports, that should not be there. If i got it right.
Here is a list of them (i'll add more if i find more):
* Server
* EnderDragon
* Enderman
* SimpleServiceManager
* StatusResponse
* Subscribe
* ClassGenerator
* BiomeArea
* BlockBuffer
* AccessorFirstStrategy
* WorldGenerator
* ClassGeneratorProviderTest